src/ManageCourses.Domain/DatabaseAccess/ManageCoursesDbContext.cs

```diff
@@ -347,5 +357,6 @@ public void Save()
         {
             SaveChanges();
         }
+
```

Comment:
personally I strip out these blank lines after the last method in a class

minor

Reply:
Agree - thanks :+1: